CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [0.11.2] - 2020-10-29
+
+### What's New
+ - Added rate limiting to logging that occurs under certain misconfigurations (#1901)
```

Comment:
minor nit on this message - the misconfiguration in this case is really a bug, and the messages are error messages, so it would be good to be specific about those two things. for example, one way we could write it is

```suggestion
 - Error messages related to a specific class of software bugs are now rate limited (#1901)
```

Reply:
i like that